README.md

```diff
@@ -104,6 +105,7 @@ Here is a list of manifests for the devices currently supported.
 | ARM Juno board| `juno.xml` | `juno_stable.xml` |
 | Raspberry Pi 3 | `rpi3.xml` | `rpi3_stable.xml` |
 | DRA7xx | `dra7xx.xml` | `dra7xx_stable.xml` |
+| AM57xx | `am57xx.xml` | N/A |
```

Comment:
An am57xx_stable.xml will be generated when the dust settles, lets add the reference here so we don't forget later. No harm in that I think.